packages/remix/tsconfig.json

```diff
@@ -5,6 +5,7 @@
 
   "compilerOptions": {
     "jsx": "react",
-    "module": "es2020"
+    "module": "es2020",
+    "ignoreDeprecations": "5.0"
```

Comment:
We won't need this unless we update the TS version right?

Reply:
that's right, and that commit has been reverted